docs/source/roadmap.md

```diff
@@ -0,0 +1,287 @@
+
+### Scientific assessment of analyzed SpaceTx data: methods comparison and meta analysis
+The SpaceTx consortium Working Group 6 is devoted to comparing the results of each method and assessing their relative strengths and weaknesses. These are largely scientific analyses enabled by the standardization of each assay's outputs by _starfish_.  Because each assay outputs an IntensityTable, Segmentation Mask, and Expression Matrix, it will be simple to label cells or spots with anatomical information represented as named 2D polygons.
+
+By enabling the creation of standardized, highly compressed outputs that are compatible with both R and Python, _starfish_ will enable working Group 6 to ask questions across all methods like "how many cells/mm2 belonging to the Sst-Chodl cluster were found in layer 4 of mouse primary visual cortex?"  or "Within the inhibitory cell class, what is the abundance and distribution of Pvalb+ and VIP+ cells in human cortex?"
+
+Finally, with a standard format to save and load Cell x Gene tables, visualization of this data will be straightforward for analysts in contributor labs and Working Group 6 members.
+
+## What is _starfish_?
+We aim for _starfish_ to be a comprehensive platform for the processing of image-based transcriptomics experiments. It should be intuitive to use, fast, and should not require extensive experience with python programming. However, at least during the initial development period, _starfish_ will require domain experience in image processing, as our target users are experts in image generation and processing.
+
+### A General Format and Object Model for Image-based Transcriptomics
+_starfish_ defines a general set of data formats (and paired python object implementations) that (1) enable efficient storage and access for a related set of images broken up into single fields of view<sup>[6](#fn6)</sup>, (2) enable storage and utilization of a codebook, which defines how sets of images are combined to decode spot patterns into biological targets, and (3) defines a pipeline recipe which can be interpreted by _starfish_'s pipeline runner to process an experiment.
+
+Currently, starfish needs images in SpaceTx Format must either be pre-aligned, such that the x-y location of each tile in a Field of View is the same. Image data that does not adhere to this requirement will not be processable with _starfish_. See [Candidate features for future releases](#candidate-features-for-future-releases) for more discussion of future directions here.
+
+### A Modular Library of Pipeline Components
+_starfish_ enables the construction of arbitrary image-based transcriptomics pipelines by implementing the union of algorithms used by SpaceTx labs while combining similar approaches into a simplified subset. These algorithms can be linearly combined to process individual fields of view, requiring no more than 16GB ram for any individual pipeline component. The set of algorithms must include:
```

Comment:
Is this 16GB reserved for starfish?  If we are trying to fit within some machine type on GS/EC2, we need to take into account what the OS and other services demand.

Reply:
Made this more ambiguous. Should be "runs on a laptop".